rust_src/src/remacs_sys.rs

```diff
@@ -81,6 +81,7 @@ extern "C" {
         depth: libc::c_int,
         ht: Lisp_Object,
     ) -> bool;
+    pub fn uniprop_table_uncompress(table: Lisp_Object, idx: u32) -> Lisp_Object;
```

Comment:
All the `uniprop_table_uncompress` stuff is unrelated.

Reply:
There should not be any uniprop changes in this PR now. I reverted all of it.

Reply:
No worries!